lib/console/schema/cluster.ex

```diff
@@ -51,6 +51,22 @@ defmodule Console.Schema.Cluster do
     end
   end
 
+  defmodule ClusterConfiguration do
```

Comment:
So I actually think we should put these in CloudSettings since it would allow them to be strictly typed for each cloud.  Perhaps we could also have this for stray stuff but biasing towards typing will keep the system generally more consistent.

You'll then need to write a bit of translation code to flush those settings into the service config